cmake/wxWidgets.cmake

```diff
@@ -31,6 +31,19 @@ SET(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${CMAKE_CURRENT_SOURCE_DIR}/cmake/Mo
 FIND_PACKAGE(wxWidgetsTB REQUIRED)
 INCLUDE("${wxWidgets_USE_FILE}")
 
+# Check version
+message(STATUS "wxWidgets ${wxWidgets_VERSION_MAJOR}.${wxWidgets_VERSION_MINOR}.${wxWidgets_VERSION_PATCH} found.")
+
+SET(TB_WX_MAJOR 3)
```

Comment:
I'm not sure if this is the place to provide the version requirement. I'd rather have it in the main CMakeLists. What do you think?

Reply:
The main file is fine. I moved them and gave them more descriptive names `TB_MIN_wxWidgets_VERSION_MAJOR`, etc